### DISCUSSION TITLE
Cannot launch earlier Bonsai versions

### DISCUSSION THREAD

**Discussion by firsttimegituser** [bonsai-rx/bonsai#1036] [General]
Tried 2.4 and 2.5 and get an error upon launching. Says  (for example)  the package "Bonsai 2.5.0 could not be found"

![image](https://user-images.githubusercontent.com/113070373/189633404-3990cb37-f16a-462a-ab24-bbba3d5d3356.png)

**Comment by bruno-f-cruz** [comment]
How did you install these versions? Can you try to do a clean install using the official releases? https://github.com/bonsai-rx/bonsai/releases

**Comment by firsttimegituser** [comment]
I downloaded from the official releases. Tried again clean install but same problem. In the window after the original error message this is what it says:

![image](https://user-images.githubusercontent.com/113070373/189635922-9dd7cc22-3e55-4363-b735-1007073da954.png)

**Comment by glopesdev** [comment]
@firsttimegituser can you check my comment on https://github.com/bonsai-rx/bonsai/discussions/1033? I think these two issues are related.

**Reply by firsttimegituser** [reply]
Regarding this inability to launch older versions. Any idea given what was discussed on the other thread? Why is saying that 2.4 and 2.5 is missing?

**Reply by glopesdev** [reply]
It's hard to say without a more detailed investigation of the machine configuration. This error message just means the application could not find those packages, but this could be for a variety of reasons, e.g. internet connection or server unavailable/blocked, corrupted cache, etc. All relevant packages are listed in both nuget.org and bonsai feed, and we haven't received these kind of reports from other users, so this makes me suspect of some local configuration issue, but would require further investigation which is hard to do without full access to the machine.

**Reply by firsttimegituser** [reply]
We have been banging our heads on this for awhile and experiments have been affected significantly so we are open to having you take a look with remote control. Should we set a meeting?
